index.js

```diff
@@ -177,16 +177,20 @@ function initialize(args) {
     // Do not make modifications to this.
     var originalPathItem = originalApiDoc.paths[openapiPath] || {};
     var pathItem = apiDoc.paths[openapiPath] || {};
-    var pathParameters = Array.isArray(pathModule.parameters) ?
-        [].concat(pathModule.parameters) :
-        [];
+    var pathParameters = pathItem.parameters || [];
+
+    // push all parameters defined in the path module to the path parameter list
+    if (Array.isArray(pathModule.parameters)) {
+      Array.prototype.push.apply(pathParameters, pathModule.parameters);
```

Comment:
Change this to:

```
[].push.apply...
```

Reply:
Just preference or any particular reason?

Reply:
Existing style + terseness + Array.prototype can be modified.

Reply:
I'll make it consistent with the rest of the codebase.

```js
Array.prototype.push = 1;
[].push === 1 // true
```

Reply:
Sadly, I remember the days of trying to decide between jQuery and http://api.prototypejs.org/language/Array/. :slightly_frowning_face: